lib/models/speaker.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:ecellapp/core/res/strings.dart';
+import 'package:equatable/equatable.dart';
+
+class Speaker extends Equatable {
+  final String comapny;
+  final String contact;
+  final String createdAt;
```

Comment:
don't add fields that are not relevant for the app. Discuss with the backend team / refer the designs and add only those fields that are needed to be displayed in the app. Also remove them from `strings.dart`